addon-test-support/assertions/deprecations.js

```diff
@@ -0,0 +1,23 @@
+import { getDeprecationsDuringCallback } from '@ember/test-helpers';
+
+export default async function deprecations(callback, expectedDeprecations) {
```

Comment:
can we merge with expectDeprecation` with similar idea as mentioned https://github.com/emberjs/ember-qunit/pull/868/files#r687868636